.mk/development.mk

```diff
@@ -69,3 +69,35 @@ undeploy-infra: undeploy-loki undeploy-grafana uninstall
 
 .PHONY: undeploy-all
 undeploy-all: undeploy-infra undeploy-sample-cr undeploy-sample-workload
+
+.PHONY: generate-flp-configuration
+generate-flp-configuration: ## Generate custom metrics configuration
+	@echo -e "\n==> Generate metrics configuration"
+	sudo rm -rf /tmp/network_definitions || true
+	sudo rm -rf /tmp/destDir || true
+	cp -r network_definitions /tmp/network_definitions
```

Comment:
please move and rename `network_definitions` folder into `/config/metrics_definitions`